testsuite/replay_verify.py

```diff
@@ -56,7 +54,9 @@
     # Skip tapos2 range
     (1_635_000_000, 1_655_000_000),
     (1_655_000_000, 1_675_000_000),
-    (1_675_000_000, sys.maxsize),
+    (1_675_000_000, 1_690_000_000),
```

Comment:
The number of ranges is hard coded here for both mainnet and testnet. Cannot add ranges just for one network.